.test/test_framework/container/tests/aws_secrets_manager_test.go

```diff
@@ -386,9 +386,13 @@ func failoverTest(t *testing.T, env *test_utils.TestEnvironment, secretName stri
 	currWriterId, err := auroraTestUtility.GetClusterWriterInstanceId("")
 	assert.Nil(t, err)
 	assert.Equal(t, currWriterId, newInstanceId)
-	assert.NotEqual(t, instanceId, newInstanceId)
 
-	test_utils.BasicCleanup(t.Name())
+	// Skip for multi-AZ b/c it simulates failover which reconnects to the original instance.
```

Comment:
```suggestion
	// Different behaviour for multi-AZ b/c it simulates failover which reconnects to the original instance.
```
Bit of a nit: but it technically isn't skipping anymore